src/main/java/com/hazelcast/commandline/member/HazelcastProcess.java

```diff
@@ -0,0 +1,21 @@
+package com.hazelcast.commandline.member;
+
+import java.io.Serializable;
+
+public class HazelcastProcess implements Serializable {
+    private String processUniqueId;
```

Comment:
Mesut also mentioned the same, I didn't use `name` since this is the unique identifier of the process in the command line. But it looks it is difficult to understand, let me change it to `name`.